cmd/kubeadm/app/discovery/file/file.go

```diff
@@ -99,7 +98,7 @@ func ValidateClusterInfo(clusterinfo *clientcmdapi.Config) (*clientcmdapi.Cluste
 		return defaultCluster, nil
 	}
 
-	fmt.Println("[discovery] Synced cluster-info information from the API Server so we have got the latest information")
+	fmt.Println("[discovery] Synced cluster-info information from the API Server so we have got the latest information.")
```

Comment:
Similar to other PR (https://github.com/kubernetes/kubernetes/pull/45708) I don't think we should add periods here.